### PR TITLE
Project structure created

### LINKED ISSUE #1: Project structure creation

### PR DESCRIPTION
**Pull request by xaberue**
New immplementations:
- #1 : Project structure created
  - Solution created
  - Libraries created
  - Dependencies added
  - First interfaces and implementations added

(#1)

### CODE REVIEW

**Review by vvillena — DISMISSED** [review]

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ChustaSoft.GamersPlatformUtils.Abstractions.Contracts
+{
+    public interface IPlatformFactory
+    {
+
+        Task<IEnumerable<IPlatform>> GetPlatforms();
```
Comment (Arthus15) [inline ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs]: Naming change to Async

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+        Task<IEnumerable<IAnalyzer>> GetAnalyzers();
```
Comment (Arthus15) [inline ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs]: Naming change to Async

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+
+        Task<IEnumerable<ICleaner>> GetCleaners();
```
Comment (Arthus15) [inline ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs]: Naming change to Async

File: GamersPlatformUtils/Implementations/PlatformFactory.cs [hunk 4]
```diff
@@ -0,0 +1,25 @@
+﻿using ChustaSoft.GamersPlatformUtils.Abstractions.Contracts;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+namespace ChustaSoft.GamersPlatformUtils.Domain.Implementations
+{
+    public class PlatformFactory : IPlatformFactory
+    {
+        public Task<IEnumerable<IAnalyzer>> GetAnalyzers()
```
Comment (Arthus15) [inline GamersPlatformUtils/Implementations/PlatformFactory.cs]: Naming change to Async

File: GamersPlatformUtils/Implementations/PlatformFactory.cs [hunk 5]
```diff
@@ -0,0 +1,25 @@
...
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<IEnumerable<ICleaner>> GetCleaners()
```
Comment (Arthus15) [inline GamersPlatformUtils/Implementations/PlatformFactory.cs]: Naming change to Async

File: GamersPlatformUtils/Implementations/PlatformFactory.cs [hunk 6]
```diff
@@ -0,0 +1,25 @@
...
+        {
+            throw new NotImplementedException();
+        }
+
+        public Task<IEnumerable<IPlatform>> GetPlatforms()
```
Comment (Arthus15) [inline GamersPlatformUtils/Implementations/PlatformFactory.cs]: Naming change to Async

**Review by Arthus15 — CHANGES_REQUESTED** [review]

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs (hunk 1 above)
Comment (xaberue) [reply ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs]: Done!

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs (hunk 2 above)
Comment (xaberue) [reply ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs]: Done!

File: GamersPlatformUtils/Implementations/PlatformFactory.cs (hunk 4 above)
Comment (xaberue) [reply GamersPlatformUtils/Implementations/PlatformFactory.cs]: Done!

File: ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs (hunk 3 above)
Comment (xaberue) [reply ChustaSoft.GamersPlatformUtils.Abstractions/Contracts/IPlatformFactory.cs]: Done!

File: GamersPlatformUtils/Implementations/PlatformFactory.cs (hunk 5 above)
Comment (xaberue) [reply GamersPlatformUtils/Implementations/PlatformFactory.cs]: Done!

File: GamersPlatformUtils/Implementations/PlatformFactory.cs (hunk 6 above)
Comment (xaberue) [reply GamersPlatformUtils/Implementations/PlatformFactory.cs]: Done!

**Review by Arthus15 — APPROVED** [review]

**Review by Arthus15 — APPROVED** [review]

**Review by vvillena — APPROVED** [review]

**Pull request merged**